app/assets/stylesheets/application.css.scss

```diff
@@ -75,6 +79,10 @@ footer {
   padding-bottom: 30px;
   padding-top: 30px;
   width: 75%;
+  max-width: 1000px;
+  input[type="file"] {
```

Comment:
Rule declaration should be preceded by an empty line<br>Avoid qualifying attribute selectors with an element.